06_gpu_and_ml/stable_diffusion/stable_diffusion_xl_turbo.py

```diff
@@ -48,11 +83,17 @@ def inference(self, image_bytes, prompt):
         init_image = load_image(Image.open(BytesIO(image_bytes))).resize(
             (512, 512)
         )
+        num_inference_steps = 4
+        strength = 0.9
+        # "When using SDXL-Turbo for image-to-image generation, make sure that num_inference_steps * strength is larger or equal to 1"
+        # See: https://huggingface.co/stabilityai/sdxl-turbo
+        assert num_inference_steps * strength >= 1
+
         image = self.pipe(
             prompt,
             image=init_image,
-            num_inference_steps=4,
-            strength=0.9,
+            num_inference_steps=num_inference_steps,
+            strength=strength,
```

Comment:
I think it'd be useful to add something at the bottom for how to run this. I.e. what command to type